feature/groups/src/main/java/com/mifos/feature/groupsList/domain/repository/GroupsListRepository.kt

```diff
@@ -0,0 +1,15 @@
+package com.mifos.feature.groupsList.domain.repository
+
+import androidx.paging.PagingData
+import com.mifos.core.objects.client.Page
+import com.mifos.core.objects.group.Group
+import kotlinx.coroutines.flow.Flow
+import rx.Observable
+
+interface GroupsListRepository {
+
+    fun getAllGroups(limit: Int): Flow<PagingData<Group>>
+
+    fun getAllDatabaseGroups(): Observable<Page<Group>>
```

Comment:
Rename `getAllLocalGroups` and can we support flow here?

Reply:
Renamed to getAllLocalGroups